features/intl-display-names.yml

```
@@ -0,0 +1,10 @@
+name: Intl.DisplayNames
+description: "The `Intl.DisplayNames` object contains consistent translations of language, region and script display names."
```

```suggestion
description: "The `Intl.DisplayNames` API translates and localizes the names of languages, regions, and scripts."
```

Suggestion to align with the other `Intl` description a bit. Not sure if "translates and localizes" is quite right. I wanted both terms to findable by search. Open to more ideas though.

---

I think this API is basically a lookup, so not sure it really "translates". In https://github.com/tc39/proposal-intl-displaynames it says about itself: "Get localized display names for languages, scripts, regions and others."

```suggestion
description: "The `Intl.DisplayNames` API provides localized names of language, region, script, and currency codes."
```

---

OK, your version works for me. 👍